scripts/minikube.sh

```diff
@@ -139,6 +139,34 @@ function validate_sidecar() {
 done
 }
 
+# install_podman_wrapper creates /usr/bin/podman.wrapper which adds /sys
+# filesystem mount points when a privileged container is started. This makes it
+# possible to map RBD devices in the container that minikube creates when
+# VM_DRIVER=podman is used.
+function install_podman_wrapper() {
+    if [[ -e /usr/bin/podman.wrapper ]]
+    then
+        return
+    fi
+
+    echo '#!/bin/sh
```

Comment:
aaah, shellcheck complains... Will address in the next update, waiting for CI jobs to progress/pass first.